File: silly_print/ruby/lib/silly_print/printers/cow.rb
module SillyPrint
  module Printers
    class Cow < Base

      def format(message)
        <<ASCII
#{message}
      \\   ^__^
       \\  (oo)\\_______
          (__)\\       )\\/\\
              ||----w |
              ||     ||

ASCII
      end

    end
  end
end

File: namespace_plugins/php/lib/SillyPrint/Printers/Cow.php
<?php

namespace SillyPrint\Printers;

class Cow extends Base
{
	function format($message)
	{
		return <<<ASCII
$message
      \   ^__^
       \  (oo)\_______
          (__)\       )\/\
              ||----w |
              ||     ||

ASCII;
	}
}
